src/deadline_worker_agent/aws/deadline/__init__.py

```diff
@@ -835,6 +835,15 @@ def record_uncaught_exception_telemetry_event(exception_type: str) -> None:
     )
 
 
+def _record_asset_sync_filesystem_event(queue_id: str, file_system: str) -> None:
+    """Calls the telemetry client to record what filesystem was used"""
+    details: Dict[str, Any] = {"queue_id": queue_id, "filesystem": file_system}
+    _get_deadline_telemetry_client().record_event(
+        event_type="com.amazon.rum.deadline.worker_agent.sync_inputs_filesystem",
```

Comment:
```suggestion
        event_type="com.amazon.rum.deadline.worker_agent.attachment_download_filesystem",
```

What about attachment_download_filesystem to match other attachment download events? `sync_inputs` is the term for the deprecating code path.